pkg/parcacol/querier.go

```diff
@@ -83,7 +88,7 @@ func (q *Querier) Labels(
 
 	err := q.engine.ScanSchema(q.tableName).
 		Distinct(logicalplan.Col("name")).
-		Filter(logicalplan.Col("name").RegexMatch("^labels\\..+$")).
+		Filter(logicalplan.Col("name").RegexMatch("^"+ColumnLabels+"\\..+$|^"+ColumnPprofLabels+"\\..+$")).
```

Comment:
the `.` in the constant won't quite behave the way we want here, it will match any character ... templating regexes is tricky I think it's easier to reason about if we just write it out in this case